components/financial_assistance/app/views/financial_assistance/applicants/_applicants.html.erb

```diff
@@ -17,7 +17,7 @@
     </div>
     <div class="col-md-2 col-sm-6 no-pd form-group-lg no-pd class-fa-household">
       <label class="static_label label-floatlabel" for="name">AGE</label>
-      <span class="field_value floatlabel form-control active-floatlabel" id="name"><%= "#{@person.age_on(Date.today)}" %></span>
+      <span class="field_value floatlabel form-control active-floatlabel" id="name"><%= "#{@person.age_on(Date.today)unless @person.dob.blank?}" %></span>
```

Comment:
can we move this logic to the model instead of the view?

Reply:
Instead of moving it to the logic I think we can just remove it completely. With the rescue in the model it's more redundant than anything now